src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -34,19 +35,23 @@ def test_my_function(self):
     Will likely be removed (or modified, at least), when a minified KBase deploy becomes available.
     Then we don't need to mock as much.
     """
+    config = ConfigTests()
+    _ee2_job_info = config.load_json_file(
+        config.get("jobs", "ee2_job_info_file")
+    )
 
     def __init__(self, token=None):
         if token is not None:
             assert isinstance(token, str)
-        self.config = ConfigTests()
         self.job_info = self.config.load_json_file(
             self.config.get("jobs", "job_info_file")
         )
-        self.ee2_job_info = self.config.load_json_file(
-            self.config.get("jobs", "ee2_job_info_file")
-        )
         self.test_job_id = self.config.get("app_tests", "test_job_id")
 
+    @property
```

Comment:
Made this attribute the property type so I could deep copy it to avoid testing bugs